_posts/2020-12-24-newpipe-0.20.7.md

```diff
@@ -0,0 +1,60 @@
+date: 2020-12-24 19:00:00
+categories: [pinned, release]
+author: Poolitzer
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+
+Merry christmas, if you celebrate that, and almost happy new year, if you use a calender at least close to the gregorian one. We want to present you a little gift as a good way to start into 2021 AD: The next release of NewPipe. It comes with a lot of improvement, fixes, and even a big new feature. So let us dive into this release post
+
+<!-- more -->
+
+## New in this version
+YouTube mixes are a feature now, and they are dropping in hot (and with a lot of bass). Opening a mix in previous versions resulted in the first video of the mix playing, nothing else. Now you see the full playlist and even a nice radio icon as an indicator. Enjoy listing to never ending lists and thank [@Stypox](Stypox) as well as [@XiangRongLin](https://github.com/XiangRongLin) for this feature.
+
+This release includes support for another feature from YouTube: meta info. The little boxes sometimes telling you that a channel belongs to a news group, or giving you further information to specific search queries - they appear in NewPipe now as well. If you do not want to see those boxes, head over to <code><nobr><img src="{{ site.baseurl }}/img/icons/baseline-settings-20px.svg" /> Settings</nobr> > <nobr><img src="{{ site.baseurl }}/img/icons/baseline-language-20px.svg" /> Content</nobr> > <nobr>Show meta info</nobr></code> and disable the display there. If you enjoy these boxes however, thank [@TobiGr](https://github.com/TobiGr) for bringing them into this release.
+
+In some kind of more personal news, we have a new domain now - [https://newpipe.net](https://newpipe.net). This makes us look more professional (we hope), but also meant the URLs needed to be updated in the app. This was done by [@TiA4f8R](https://github.com/TiA4f8R).
+## Bugs, be gone!
+We aim for the best NewPipe with every release we do. Because of that, we improve on issues in every update. This time around, we fixed an issue where the app would crash when you tried to delete a video from your playlist while the app was refreshing your subscriptions. This was discovered by [@gasull](gasull), reproduced by [@chilliger](https://github.com/chilliger) and fixed by [@hlloreda](https://github.com/hlloreda). Thanks to all three of them. 
+
+In another attempt to improve the ReCaptcha experience, [@Stypox](https://github.com/Stypox) fixed a situation where only a blank page was shown and is likely very exhausted with that topic at this point. We thank him massively for taking up this issue time and time again.
+
+Talking about time, sharing URLs from any service except YouTube now doesn't try to add a timestamp to the URL of the video. This broke the URL on any other service. In the future we hope to support the time stamps of every platform, for now, you get non timed URLs that work. This was provided by [@B0pol](https://github.com/B0pol), thank you!
```

Comment:
```suggestion
Talking about time, sharing URLs from any service except YouTube now doesn't try to add a timestamp to the URL of the video. This broke the URL on any other service. In the future we hope to support the timestamps of every platform, for now, you get non-timed URLs that work. This was provided by [@B0pol](https://github.com/B0pol), thank you!
```
From my quick search, it seems that both "non-timed" and "nontimed" work. Choose whichever one you want.

Reply:
more - is all I want (for christmas)